.travis.yml

```diff
@@ -21,9 +24,17 @@ branches:
   only:
     - master
 before_install: |
-  # Install bundler < 2.0, see https://docs.travis-ci.com/user/languages/ruby/
-  gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
-  gem install bundler -v '< 2'
+  RUBY_MAJOR=$(echo "$TRAVIS_RUBY_VERSION" | cut -d. -f1)
+  RUBY_MINOR=$(echo "$TRAVIS_RUBY_VERSION" | cut -d. -f2)
+
+  if (( $RUBY_MAJOR >= 3 && $RUBY_MINOR >= 2)); then
+    # use travis default bundler
+  else
+    # Install bundler < 2.0, see https://docs.travis-ci.com/user/languages/ruby/
+    gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
+    gem install bundler -v '< 2'
+  fi 
```

Comment:
I think we can simply use the default bundler under Travis now that Ruby 2.2 is dropped